packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -175,6 +176,7 @@ export const registerPostTypeSchema =
 			postTypeConfig.supports?.comments && commentStatusField,
 			templateField,
 			passwordField,
+			contentPreviewField,
```

Comment:
I haven't checked all the code yet, but if we want this field we should check for `editor (content)` support of the post type.

Reply:
I added a check for viewable posts with editor support.